toolkit/scripts/toolchain/container/toolchain-sha256sums

```diff
@@ -24,7 +24,7 @@ a6bdd7d1b31266d11c4f4de6c1b748d4607ab0231af5188fc2533d0ae2438fec  diffutils-3.8.
 e72051e6a3e06b236d19eed56368117b745ca1e1a27bdc50fd51aa375bea6509  dmxproto-2.3.1.tar.bz2
 f9b55476def44fc7c459b2537d17dbc731e36ed5d416af7ca0b1e2e676f8aa04  dri2proto-2.8.tar.bz2
 01be49d70200518b9a6b297131f6cc71f4ea2de17436896af153226a774fc074  dri3proto-1.0.tar.bz2
-9bf7200d2737ed13f50a080af285c11529f91b088d84ecb71aae9fac58a8fbee  e2fsprogs-1.44.6.tar.gz
+7524520b291e901431ce59ea085955b601126de371bf3cfc0f5e4fad78684265  e2fsprogs-1.46.4.tar.gz
```

Comment:
Weird that it's in this list but not build in the first stage of the toolchain but that's out of scope for this change, so LGTM.;)

Reply:
I think this is a good opportunity to clean this up - it doesn't look like it's needed. Can you remove this from both files?

Reply:
Cleaned out e2fsprogs from the two files.